Dockerfile

```diff
@@ -15,7 +15,7 @@ COPY . .
 
 RUN CGO_ENABLED=1 GOOS=linux go build -a -installsuffix cgo -o nightfall_code_scanner ./cmd/nightfalldlp/
 
-FROM alpine:3.8
+FROM alpine:latest
```

Comment:
I am wondering if we should use `latest` as the tag. If we use this then we will always be grabbing the newest alpine image. So we can eventually start pulling images that have a major version increase without having tested it. Not sure if this is desired? 

At one point I thought you could use the tag `3.x.x` or `3.x` and this would always get the latest version on major version `3`. But cannot seem to get this to work locally when pulling the alpine image.